client/src/main/java/gui/printers/FieldPrinter.java

```diff
@@ -1,15 +1,23 @@
 package gui.printers;
 
 import connection.ServerConnection;
+import gui.data.Field;
+import gui.data.FieldState;
 import gui.instance.ClientAppRunner;
+import gui.recievers.ShotResult;
+import javafx.collections.ObservableList;
 import javafx.scene.control.Button;
 import javafx.scene.layout.GridPane;
+import model.Coordinates;
 
 /**
  * This class is made to help with printing fields as buttons on enemy board.
  */
 public class FieldPrinter {
 
+    private static final String WATER_STYLE = "water-button";
```

Comment:
6 fields ??? Wrap it

No line separators, for common fields

Reply:
Wraped in enum.